eos/form-factors/analytic-d-to-pi.cc

```diff
@@ -0,0 +1,2174 @@
+                    + r22 * (10.0 - 22.0 * r1 + 12.0 * r12 + 2.0 * r13 - 2.0 * r14)
+                    + r23 * (-6.0 + 14.0 * r1 - 10.0 * r12 + 2.0 * r13)
+                    + r24 * (1.0 - 2.0 * r1 + r12);
+
+                const double ca40 = 42.0 * r1 - 126.0 * r12 + 140.0 * r13 - 70.0 * r14 + 15.0 * r15 - r16
+                    + r2 * (-42.0 + 210.0 * r12 - 280.0 * r13 + 135.0 * r14 - 24.0 * r15 + r16)
+                    + r22 * (126.0 - 210.0 * r1 + 150.0 * r13 - 75.0 * r14 + 9.0 * r15)
+                    + r23 * (-140.0 + 280.0 * r1 - 150.0 * r12 + 10.0 * r14)
+                    + r24 * (70.0 - 135.0 * r1 + 75.0 * r12 - 10.0 * r13)
+                    + r25 * (-15.0 + 24.0 * r1 - 9.0 * r12)
+                    + r26 * (1.0 - r1);
+                const double ca4r1 = 42.0 * r1 - 168.0 * r12 + 266.0 * r13 - 210.0 * r14 + 85.0 * r15 - 16.0 * r16 + r17
+                    + r2 * (-42.0 + 42.0 * r1 + 210.0 * r12 - 490.0 * r13 + 415.0 * r14 - 159.0 * r15 + 25.0 * r16 - r17)
+                    + r22 * (126.0 - 336.0 * r1 + 210.0 * r12 + 150.0 * r13 - 225.0 * r14 + 84.0 * r15 - 9.0 * r16)
+                    + r23 * (-140.0 + 420.0 * r1 - 430.0 * r12 + 150.0 * r13 + 10.0 * r14 - 10.0 * r15)
+                    + r24 * (70.0 - 205.0 * r1 + 210.0 * r12 - 85.0 * r13 + 10.0 * r14)
+                    + r25 * (-15.0 + 39.0 * r1 - 33.0 * r12 + 9.0 * r13)
+                    + r26 * (1.0 - 2.0 * r1 + r12);
+
+                return -6.0 / (r1 * r1 * power_of<7>(r1 - r2)) * (
+                            power_of<4>(r1 - r2) * (ca00 * r1 + ca0r1 * L1mr1)
+                            + 6.0 * power_of<2>(r1 - r2) * (ca20 * r1 + ca2r1 * L1mr1) * a2pi
+                            + 15.0 * (ca40 * r1 + ca4r1 * L1mr1) * a4pi
+                        );
+
+            };
+            std::function<double (const double &)> integrand(
+                [&] (const double & r2) -> double
+                {
+                    // _select_weight:
+                    //  0.0 -> regular integral
+                    //  1.0 -> integral of derivative w.r.t. -1/M^2
+                    const double weight = (1.0 - _select_weight) + _select_weight * mc2 * r2;
+
+                    return (T1tiltw2theta1mrho(r1, r2) + T1tiltw2thetarhom1(r1, r2) + T1tiltw2delta(r1, r2))
+                       * weight * std::exp(-mc2 * r2 / _M2);
+                }
+            );
+
+            static const double eps = 1e-12;
+
+            return mc2 * fpi * integrate<GSL::QAGS>(integrand, 1.0 + eps, s0tilD(q2) / mc2, config);
+        }
+
+        double Ftil_nlo_tw3(const double & q2, const double _M2, const double & _select_weight = 0.0) const
+        {
+            // Reminder: q2 is the kinematic variable associated with the momentum
+            // transfer, while s is the kinematic variable in which the function is
+            // analytically continued. See also comment at beginning of Appendix B
+            // of [KMMO2009], p. 21.
```

Comment:
Ditto, here and in other places in the code

Reply:
My point was that p. 21 of `[KMMO:2009A]` (please use this reference template also in the comments) is not appendix B and doesn't have the relevant comment you mention.